src/main/webapp/app/shared/table/VusTable.tsx

```diff
@@ -0,0 +1,264 @@
+
+    await handleFirebaseUpdate(firebaseVusPath, updateObject);
+  }
+
+  const columns: SearchColumn<VusTableData>[] = [
+    {
+      Header: VUS_NAME,
+      accessor: 'name',
+      onFilter(data, keyword) {
+        return data.name.toLowerCase().includes(keyword);
+      },
+    },
+    {
+      Header: LAST_EDITED_BY,
+      accessor: 'time.by.name',
+      onFilter(data, keyword) {
+        return data.time.by.name.toLowerCase().includes(keyword);
+      },
+    },
+    {
+      Header: LAST_EDITED_AT,
+      id: LAST_EDITED_AT,
+      accessor: 'time.value',
+      Cell(cell: { original: VusTableData }) {
+        const time = cell.original.time.value;
+        return <TextFormat value={new Date(time)} type="date" format={APP_DATETIME_FORMAT} />;
+      },
+    },
+    {
+      Header: LATEST_COMMENT,
+      accessor: 'name_comments',
+      Cell(cell: { original: VusTableData }) {
+        const latestComment = cell.original.name_comments ? getMostRecentComment(cell.original.name_comments).content : '';
+
+        return (
+          <span>{latestComment.length <= MAX_COMMENT_LENGTH ? latestComment : `${latestComment.slice(0, MAX_COMMENT_LENGTH)}...`}</span>
+        );
+      },
+    },
+    {
+      id: 'actions',
+      Header: 'Actions',
+      Cell(cell: { original: VusTableData }) {
+        return (
+          <Container>
+            <Row>
+              <CommentIcon
+                id={cell.original.uuid}
+                key={cell.original.uuid}
+                comments={cell.original.name_comments}
+                onCreateComment={async content => {
+                  await handleCreateComment(cell.original.uuid, content);
+                }}
+                onDeleteComments={async indices => {
+                  await handleDeleteComments(cell.original.uuid, indices);
+                }}
+                onResolveComment={async index => {
+                  await handleSetCommentResolved(cell.original.uuid, index, 'true');
+                }}
+                onUnresolveComment={async index => {
+                  await handleSetCommentResolved(cell.original.uuid, index, 'false');
+                }}
+              />
+              <Button
+                className="mx-2"
+                size="sm"
+                color="primary"
+                onClick={async () => {
+                  await handleRefresh(cell.original.uuid);
+                }}
+              >
+                <FontAwesomeIcon icon={'sync'} />
+              </Button>
+              <Button
+                className="mr-2"
+                size="sm"
+                color="danger"
+                onClick={() => {
+                  currentActionVusUuid.current = cell.original.uuid;
+                  setShowConfirmModal(true);
+                }}
+              >
+                <FontAwesomeIcon icon={'trash'} />
+              </Button>
+            </Row>
+          </Container>
+        );
+      },
+    },
+  ];
+
+  return vusData ? (
```

Comment:
Do we have a ticket on allowing user to add VUS? If not, we need to add that in another PR.
At least the header of the table should be available to all genes no matter vus available.

Reply:
We do not yet. Maybe we can discuss in standup and make that edit when we add the 'Add VUS` button?